Financial-Times/alphaville-es-interface-service
# Remove wordpress-json-api healthcheck

Powering down the old production Alphaville Wordpress stack ahead of full decommissioning has flushed out a few failing healthchecks on Heimdall:
https://heimdall.ftops.tech/system?returnTo=%2Foperations&code=ftalphaville#monitored-checks-list

As we plan to decommission Alphaville Wordpress, I'm guessing these healthchecks aren't required any more - this PR removes the `wordpress-json-api` healthcheck.

However, it is worth noting that I do not know:
- this system
- if this is the correct way to remove healthchecks
- if further clean up work is required
- how to test or deploy this

I've marked this PR as draft in case it's helpful as a starting point for cleaning things up, but please double check everything in case I've made a mistake or this is the wrong approach. 🙂 

Linked to https://github.com/Financial-Times/alphaville-blogs/pull/77
